spec/models/collection_spec.rb

```diff
@@ -44,7 +46,17 @@
         allow(@collection).to receive(:members).and_return([file, file, file])
         allow(ActiveFedora::SolrService).to receive(:query).with(query, args).and_return(documents)
       end
-      it { is_expected.to eq 99 }
+
+      context "when saved" do
+        before do
+          allow(@collection).to receive(:new_record?).and_return(false)
+        end
+        it { is_expected.to eq 99 }
+      end
+
+      context "when not saved" do
+        it { expect {subject}.to raise_error "Collection must be saved to query for bytes" }
```

Comment:
If the `is_expected` syntax doesn't work (weird!), I'd say tweak the syntax a bit here:

```ruby
it "raises an error" do
  expect { subject }.to raise_error "Collection must be saved to query for bytes"
end
```

Because the "it { expect" syntax isn't used anywhere else in Sufia.